torch/distributions/uniform.py

```diff
@@ -40,13 +40,16 @@ def stddev(self):
     def variance(self):
         return (self.high - self.low).pow(2) / 12
 
-    def __init__(self, low, high):
+    def __init__(self, low, high, validate_args=None):
         self.low, self.high = broadcast_all(low, high)
+
+        if (self._validate_args or validate_args) and not torch.lt(self.low, self.high).all():
```

Comment:
It would be simpler if we could do the `super` call, it would *always* set `self.validate_args` (taking into account both the arg and the default), and then you'd use this to implement checks